README.md

```diff
@@ -83,19 +83,19 @@ Orientation.HORIZONTAL
 
 Add the JitPack repository to your root build.gradle
 ```
-	allprojects {
-		repositories {
-			...
-			maven { url 'https://jitpack.io' }
-		}
+allprojects {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
 	}
```

Comment:
In order to keep indentation consistent can you please update line 91.  Thanks!

Reply:
Done